plugins/by-name/wezterm/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "wezterm";
+  originalName = "wezterm.nvim";
+  package = "wezterm-nvim";
```

Comment:
> `error: vimPlugins.wezterm-nvim cannot be found in pkgs`

Looks like this [is in nixos-unstable now](https://search.nixos.org/packages?channel=unstable&show=vimPlugins.wezterm-nvim&from=0&size=50&sort=relevance&type=packages&query=wezterm-nvim). Rebasing on our latest `main` branch should bring a more up-to-date lock file and resolve this issue :upside_down_face:

Reply:
Should be good now

Reply:
We still have `error: vimPlugins.wezterm-nvim cannot be found in pkgs`, so I guess we the most recent lock bump was slightly too old.

#2610 should resolve :crossed_fingers: